docs/0_index.md

```diff
@@ -12,12 +12,12 @@ to validate accounts by email, domain or group.
 
 **Note:** This repository was forked from [bitly/OAuth2_Proxy](https://github.com/bitly/oauth2_proxy) on 27/11/2018.
 Versions v3.0.0 and up are from this fork and will have diverged from any changes in the original fork.
-A list of changes can be seen in the [CHANGELOG](CHANGELOG.md).
+A list of changes can be seen in the [CHANGELOG]({{ site.gitweb }}/CHANGELOG.md).
```

Comment:
Is this a typo? :sweat_smile:

Reply:
Is what a typo? gitweb is set in the [config](https://github.com/pusher/oauth2_proxy/pull/241/files#diff-b09bbff3e688a10b35f7de810d65c28eR21)

Reply:
Sorry, it's not correctly showing the markdown for me so I was unsure if this was a typo: 

![image](https://user-images.githubusercontent.com/3449337/62982004-beebec80-bddf-11e9-80ee-268021310aec.png)

Reply:
It's using Jekyll syntax, for https://pusher.github.io/oauth2_proxy. There's already things that aren't supported by github markdown, like if you see the "table" at the top of the page.